lib/utils/tsv_reader.rb

```diff
@@ -40,12 +42,12 @@ def records
       # For each line, get a copy of header_hash and populate it using header_index
       # so that cols_hash = {ocn: 11, local_id: i11}
       @inf.each_line do |line|
-        yield line_to_hash(line.strip)
+        yield line_to_hash(line.chomp)
```

Comment:
the changes here are to ensure we can handle trailing empty columns (not sure whether we'd expect to see that in practice or not)

Reply:
yeah that's a change for the better